scene/gui/code_edit.cpp

```diff
@@ -1734,7 +1734,10 @@ void CodeEdit::fold_all_lines() {
 }
 
 void CodeEdit::unfold_all_lines() {
-	_unhide_all_lines();
+	for (int i = 0; i < get_line_count(); i++) {
+		unfold_line(i);
+	}
+	queue_redraw();
```

Comment:
You shouldn't need to `queue_redraw` here since `unfold_line` does handle it.

`queue_redraw` puts the `CanvasItem` in the queue, see this for details: https://docs.godotengine.org/en/stable/classes/class_canvasitem.html#class-canvasitem-method-queue-redraw

Reply:
`CodeEdit::fold_all_lines()` has the same issue, though it ultimately doesn't matter. The items is only queued once (otherwise doing it in the loop via `(un)fold_line` would be a problem as well).

Reply:
Removed the `queue_redraw` call at the end of `unfold_all_lines`